packages/web-components/src/components/va-select/va-select.tsx

```diff
@@ -3,13 +3,21 @@ import {
   Element,
   Event,
   EventEmitter,
+  forceUpdate,
   Host,
   Prop,
   State,
   h,
 } from '@stencil/core';
+import i18next from 'i18next';
+import { Build } from '@stencil/core';
 import { getSlottedNodes } from '../../utils/utils';
 
+if (Build.isTesting) {
+  // Make i18next.t() return the key instead of the value
+  i18next.init({ lng: 'cimode' });
```

Comment:
We could either remove this or add a test that checks for the `required` key.

Reply:
This is the last thing I think we should address before merging. Right now it doesn't look like we need it since there are no changes to the e2e.ts file - should we remove it or add a test where it will be necessary?